File: 0708/708.c
int insertIntoASortedCircularLinke(struct TreeNode* root) {
    if (!root) return 0;
    int l = insertIntoASortedCircularLinke(root->left);
    int r = insertIntoASortedCircularLinke(root->right);
    return 1 + (l > r ? l : r);
}

File: 0708/708.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var insertIntoASortedCircularLinke = function(root) {
    if (!root) return 0;
    return 1 + Math.max(insertIntoASortedCircularLinke(root.left), insertIntoASortedCircularLinke(root.right));
};